gtest_parallel_tests.py

```diff
@@ -114,56 +114,50 @@ def run(self):
 
 class TestTaskManager(unittest.TestCase):
   def setUp(self):
-    self.times = TimesMock()
-    self.logger = LoggerMock()
-    self.test_results = TestResultsMock()
-
-    self.test_data = [
-        # Passing task
-        (('fake_binary', 'Fake.PassingTest'), {
+    self.passing_task = (
+        ('fake_binary', 'Fake.PassingTest'), {
             'runtime_ms': [10],
             'exit_code': [0],
             'last_execution_time': [10],
-        }),
-        # Fails once, then succeeds
-        (('another_binary', 'Fake.Test.FailOnce'), {
+        })
+    self.failing_task = (
+        ('fake_binary', 'Fake.FailingTest'), {
+            'runtime_ms': [20, 30, 40],
+            'exit_code': [1, 2, 3],
```

Comment:
I think you can just use 1 for failure here, there's no obvious thing that's verified by these codes being different.